pkg/interface/contacts/src/js/api.js

```diff
@@ -69,7 +69,14 @@ class UrbitApi {
   }
 
   contactCreate(path, ships = []) {
-    return this.contactViewAction({ create: { path, ships }});
+    return this.contactViewAction({
+      create: {
+        path,
+        ships,
+        title: '',
+        description: ''
```

Comment:
Do we not want to let the user input these?

Reply:
We do, but I didn’t want to do that work in this PR